matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/account/DeactivateAccountTest.kt

```diff
@@ -39,10 +39,8 @@ import kotlin.coroutines.resume
 @FixMethodOrder(MethodSorters.JVM)
 class DeactivateAccountTest : InstrumentedTest {
 
-    private val commonTestHelper = CommonTestHelper(context())
-
     @Test
-    fun deactivateAccountTest() {
+    fun deactivateAccountTest() = runSessionTest(context()) { commonTestHelper ->
```

Comment:
I am wondering if this test will work since at the end we have
`// No need to close the session, it has been deactivated`

Reply:
good catch, the test is failing. I added a try/catch in cleanup session. Not sure we can do much?
It's throwing a generic RuntimeException, can't do much filtering :/ 
WDYT

Reply:
I think for this particular test we cannot use `runSessionTest`, or maybe add an optional parameter `cleanSessionAtEndOfTest`, default to `true`, that will be `false` here. Is it possible?